plugins/node.d/http_loadtime

```diff
@@ -1,5 +1,4 @@
-#!@@BASH@@
-# -*- sh -*-
+#!/usr/bin/env bash
```

Comment:
I don't like to use `/usr/bin/env`in the source from official plugins. 
We just use `/bin/bash` or `/bin/sh`. This is much cleaner and let package maintainers handle the eventual change in PATH.